src/utils/BlockUtils.ts

```diff
@@ -101,7 +106,7 @@ export class BlockFinder {
    * @notice Gets the latest block whose timestamp is <= the provided timestamp.
    * @param {number} timestamp timestamp to search.
    */
-  public async getBlockForTimestamp(timestamp: number | string): Promise<Block> {
+  public async getBlockForTimestamp(timestamp: number | string, hints: BlockFinderHint): Promise<Block> {
```

Comment:
We should update the doc comment

Reply:
Good call: https://github.com/across-protocol/sdk-v2/pull/465/commits/d2bfc9bb16dd977dab2d30fa940234a82f3debb4